pkg/networking/kindnetd.go

```diff
@@ -0,0 +1,58 @@
+package networking
+
+import (
+	"fmt"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/yaml"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+)
+
+type Kindnetd struct{}
```

Comment:
curious why kindnetd instead of kindnet?

Reply:
thats what they call it upstream?